agent/engine/docker_task_engine.go

```diff
@@ -313,10 +313,10 @@ func (engine *DockerTaskEngine) monitorExecAgentRunning(ctx context.Context,
 	status, err := engine.execCmdMgr.RestartAgentIfStopped(ctx, engine.client, task, c, dockerID)
 	if err != nil {
 		seelog.Errorf("Task engine [%s]: Failed to restart ExecCommandAgent Process for container [%s]: %v", task.Arn, dockerID, err)
-		mTask.emitManagedAgentEvent(mTask.Task, c)
+		mTask.emitManagedAgentEvent(mTask.Task, c, "Task engine unable to restart the Execute Command Agent")
 	}
 	if status == execcmd.Restarted {
-		mTask.emitManagedAgentEvent(mTask.Task, c)
+		mTask.emitManagedAgentEvent(mTask.Task, c, "Task engine restarted the Execute Command Agent")
```

Comment:
Since this reason will be the customer facing reason, I wonder if "Task engine" is relevant. Would "Exec Command Agent Restarted" be more appropriate?